ci/terraform/oidc/authdev1.tfvars

```diff
@@ -37,6 +37,18 @@ PirNO+f1KP1HGi7oYdoiFgVXejoUmpxKXeQRtOF6PgTBkrRMzmI51YFLDQ==
 -----END PUBLIC KEY-----
 EOT
 
+ipv_authorization_public_key = <<-EOT
```

Comment:
We should have different public keys for these envs I think. Think another argument for leaving these envs be for a bit and just focusing on dev